src/openrct2/rct2/S6Importer.cpp

```diff
@@ -642,6 +642,30 @@ namespace RCT2
             }
         }
 
+        void FixWater() const
+        {
+            if (!_isScenario)
+            {
+                return;
+            }
+            if (String::Equals(_s6.ScenarioFilename, "Infernal Views.SC6", true)
+                || String::Equals(_s6.ScenarioFilename, "infernal views.sea", true))
+            {
+                auto surfaceElement = MapGetSurfaceElementAt(TileCoordsXY{ 45, 62 }.ToCoordsXY());
+
+                surfaceElement->SetWaterHeight(96);
+            }
+            else if (
+                String::Equals(_s6.ScenarioFilename, "Six Flags Holland.SC6")
```

Comment:
This one lacks the last argument. It is safe to assume the casing will match?

Reply:
Yes, that one. Did you leave it out deliberately?

Reply:
i did not deliberately leave it out, no. it did build correctly without it, but again i can add it in if needed.